editor/src/core/shared/element-path-tree.ts

```diff
@@ -25,57 +31,160 @@ export function elementPathTree(
   }
 }
 
-function emptyElementPathTree(path: ElementPath, pathString: string): ElementPathTree {
-  return elementPathTree(path, pathString, [])
+export type ElementPathTrees = { [key: string]: ElementPathTree }
+
+export function buildTree(metadata: ElementInstanceMetadataMap): ElementPathTrees {
+  const elementPaths = Object.values(metadata).map((m) => m.elementPath)
+  if (
+    elementPaths.length === 0 ||
+    elementPaths[0].parts.length < 1 ||
+    elementPaths[0].parts[0].length < 1
```

Comment:
`isEmptyPath` should do the trick here https://github.com/concrete-utopia/utopia/blob/4130f42930a17f44249d00f2a1b93d83dbdb2f00/editor/src/core/shared/element-path.ts#L218